src/setup.rs

```diff
@@ -118,11 +119,18 @@ where
     // Create query executor
     let query_executor = ExecutorImpl::new(config.query.clone());
 
+    // Config limiter
+    let limiter =  Limiter::new(
```

Comment:
How about 
```
Limiter::new(config.limiter.clone())
```
As I see, this way may be more  easy to add more items into `limiter` in the future.

Reply:
Good idea.